src/pages/Immigration/ImmigrationExit.tsx

```diff
@@ -0,0 +1,96 @@
+import FadeTo from '../../components/Scene/FadeTo'
+import { black } from '../../config/colors'
+import useStyles from './ImmigrationExit.styles'
+import { useNavigate } from 'react-router-dom'
+import { useContext, useEffect, useState } from 'react'
+import { MetadataContext } from '../../context/Metadata/MetadataContext'
+import { Trait } from '../../interface/metadata'
+import doFetch from '../../utils/doFetch'
+import { host } from '../../config/api'
+import { WardrobeRoute } from '../routes'
+import Button from '../../components/Button/Button'
+
+function ImmigrationExit() {
+	const classes = useStyles()
+	const { metadata } = useContext(MetadataContext)
+	const [peepImage, setPeepImage] = useState('')
+	const [playAnimation, setPlayAnimation] = useState(false)
```

Comment:
`playAnimation` is unused?

Reply:
I like that they are walking at the same time the fade is happening. Think we can remove this state